lms/djangoapps/courseware/tests/test_tabs.py

```diff
@@ -351,10 +345,6 @@ def setUp(self):
         super().setUp()
 
         self.course = CourseFactory.create()
-        self.instructor_tab = ItemFactory.create(
```

Comment:
Apparently not!? It tried to create an `instructor` xblock type, which doesn't exist. Old Mongo would just make whatever you wanted. Split complains if it is given an unknown xblock type. Since nothing seemed to care about this tab... I just removed it. I dunno what the intent was.